framework/generated/khronos_generators/vulkan_generators/vulkan_stype_util_generator.py

```diff
@@ -55,13 +53,10 @@ def beginFile(self, gen_opts):
             template <typename T> VkStructureType GetSType() = delete;'''), file=self.outFile)
         self.newline()
 
-    def genStruct(self, typeinfo, typename, alias):
-        if not alias and typename not in self.missing_stypes:
-            stype = self.make_structure_type_enum(typeinfo, typename)
-            if stype:
-                write(f'template <> constexpr VkStructureType GetSType<{typename}>(){{ return {stype}; }}', file=self.outFile)
-
     def endFile(self):
+        for struct in self.all_struct_members.keys():
+            if struct in self.struct_type_names:
```

Comment:
Why wouldn't all_struct_members.keys() be the same as struct_type_names?  Aliasing? Blacklist?  Maybe a comment?

Reply:
This is the "struct_type_names" enum which would have struct types for some structures such as VkBaseInStruct.  There are several other structures that also do not have one so that is why.